file/reader.go

```diff
@@ -54,6 +59,10 @@ func Get(fileContent *Content, opt RenderConfig) (*utils.KongRawState, error) {
 	builder.currentState = opt.CurrentState
 	builder.kongVersion = opt.KongVersion
 
+	if fileContent.Transform != nil && !*fileContent.Transform {
+		return nil, fmt.Errorf("_transform: false is not supported")
```

Comment:
(similarly to another PR in KIC where we discussed this)

let's export a named error value:

```go
  var ErrorTransformFalseNotSupported = fmt.Errorf("...")
```

and use it in tests instead of diffing the error text

Reply:
93af3c1d7f79dccd66e9661d855eea08e2791121